pkg/resources/management.cattle.io/v3/setting/validator_test.go

```diff
@@ -63,75 +65,55 @@ func (t *retentionTest) toOldSetting() ([]byte, error) {
 	})
 }
 
+const iso8601Date = "20240108T000000Z"
+
+var rfc3339Date = time.Now().Truncate(time.Second).Format(time.RFC3339)
+
 var retentionTests = []retentionTest{
-	{
-		setting: "disable-inactive-user-after",
-		value:   "",
-		allowed: true,
-	},
-	{
-		setting: "delete-inactive-user-after",
-		value:   "",
-		allowed: true,
-	},
-	{
-		setting: "user-last-login-default",
-		value:   "",
-		allowed: true,
-	},
-	{
-		setting: "user-retention-cron",
-		value:   "",
-		allowed: true,
-	},
-	{
-		setting: "disable-inactive-user-after",
-		value:   "2h30m",
-		allowed: true,
-	},
-	{
-		setting: "delete-inactive-user-after",
-		value:   setting.MinDeleteInactiveUserAfter.String(),
-		allowed: true,
-	},
-	{
-		setting: "user-last-login-default",
-		value:   "2024-01-08T00:00:00Z",
-		allowed: true,
-	},
-	{
-		setting: "user-retention-cron",
-		value:   "* * * * *",
-		allowed: true,
-	},
-	{
-		setting: "disable-inactive-user-after",
-		value:   "1w",
-	},
-	{
-		setting: "delete-inactive-user-after",
-		value:   "2d",
-	},
-	{
-		setting: "user-last-login-default",
-		value:   "foo",
-	},
```

Comment:
I might have missed it, but it looks like we removed this test. Can you re-add it?